test/cpp/lazy/test_lazy_ops_util.cpp

```diff
@@ -1,8 +1,8 @@
 #include <test/cpp/lazy/test_lazy_ops_util.h>
 
 #include <torch/csrc/lazy/backend/lowering_context.h>
-#include <torch/csrc/lazy/ts_backend/ops/device_data.h>
 #include <torch/csrc/lazy/core/ir_dump_util.h>
+#include <torch/csrc/lazy/core/ir_nodes.h>
```

Comment:
as to why we need a new one is because its autogenerated. But you're right. It may also also introduce a circular include.

I think I had a slightly different intention for this autogenerated file when I first made it. `ir_builder.h` is a better name for what it currently does